persistence/chain.json

```diff
@@ -36,25 +36,25 @@
   },
   "codebase": {
     "git_repo": "https://github.com/persistenceOne/persistenceCore",
-    "recommended_version": "v7.0.0",
+    "recommended_version": "v9.1.1",
     "compatible_versions": [
-      "v7.0.0"
+      "v9.1.1"
     ],
     "binaries": {
-      "linux/amd64": "https://github.com/persistenceOne/persistenceCore/releases/download/v7.0.0/persistenceCore-v7.0.0-linux-amd64.tar.gz"
+      "linux/amd64": "https://github.com/persistenceOne/persistenceCore/releases/download/v9.1.1/persistenceCore-v9.1.1-linux-amd64.tar.gz"
     },
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/persistenceOne/genesisTransactions/master/core-1/final_genesis.json"
     },
     "versions": [
       {
-        "name": "v7.0.0",
-        "recommended_version": "v7.0.0",
+        "name": "v9.1.1",
```

Comment:
Please add a new version object instead of writing over the previous version

Reply:
thanks i am just going to close this PR and open a new one, quite some has to be updated